include/php.sh

```diff
@@ -54,12 +54,19 @@ configure_php_ini()
 		fi
 	fi
 
+	tell_status "getting the timezone"
+	TZ=`md5 -q /etc/localtime`
+	TIMEZONE=`find /usr/share/zoneinfo -type f | xargs md5 -r | grep  $TZ | awk '{print $2}' |cut -c21- `
+
+	tell_status "Setting TIMEZONE to :  $TIMEZONE "
```

Comment:
I feel like this should have a default, so that if $TIMEZONE is empty, it should default to GMT or 'America/Los_Angeles'.